src/modules/httptrap.c

```diff
@@ -418,6 +417,15 @@ httptrap_yajl_cb_string(void *ctx, const unsigned char * stringVal,
     json->saw_complex_type |= HT_EX_VALUE;
     return 1;
   }
+  else if(json->last_special_key == HT_EX_TS) {
+    uint64_t ts = strtoull(stringVal, NULL, 10);
```

Comment:
Do we care if there is garbage there?  Should we be checking if this returns 0, and returning 0 as well, rather than going forward with a 0 timestamp?

Reply:
All arguable.  A _ts of 0 will eventually get treated correctly, so that seems like a non-issue.

I'm fine making this strict that the string be completely parsable, it's a change from elsewhere though.